homeassistant/components/sensor/zwave.py

```diff
@@ -53,6 +53,14 @@ class ZWaveSensor(zwave.ZWaveDeviceEntity):
     def __init__(self, value):
         """Initialize the sensor."""
         zwave.ZWaveDeviceEntity.__init__(self, value, DOMAIN)
+        self._state = None
```

Comment:
You don;t need to set to None - you are calling update_properties to init those values